bamazonCustomer.js

```diff
@@ -0,0 +1,91 @@
+var mysql = require("mysql");
+var inquirer = require("inquirer");
+
+var data;
+
+var connection = mysql.createConnection({
+  host: "localhost",
+  port: 3306,
+  user: "root",
+  password: "eqk8S8GB6fj8",
```

Comment:
It would be prudent to give the user the ability to set these for themselves if they have different cofig setup for their mysql, something like:
```javascript
password: process.env.MYSQL_PASSWORD || "eqk8S8GB6fj8"